test/test_gesv.cc

```diff
@@ -255,7 +255,8 @@ void test_gesv_work(Params& params, bool run)
 
     double gflop;
     if (params.routine == "gesv"
-        || params.routine == "gesvMixed"
+        || params.routine == "gesv_nopiv"
```

Comment:
As Neil mentioned in a previous PR, the `_nopiv` is trimmed from the `params.routine`. Therefore, this test on `_nopiv` should be removed.

Reply:
Fixed.